source/documentation/joining-our-team/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+owner_slack: "#data-catalogue"
+title: Joining Our Team
+last_reviewed_on: 2024-03-27
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## General Information
+
+- Our team have a stand up each morning at 10:45.
+- We plan our sprints using GitHub Projects.
+- We have our team Retrospective every 2 weeks at the end of each sprint.
+
+## Tools
+
+### Communication
+
+Our main chat platform is Slack, and this is our preferred method of communication.
+
+We use Slack huddles in the [#data-catalogue channel](https://moj.enterprise.slack.com/archives/C03QZ776JVA)
+for pairing or direct calls.
+
+Developers link PR requests in the [#dpl-developers channel](https://moj.enterprise.slack.com/archives/C05AABSKFLJ)
+
+Our agile ceremonies take place in Google Meet,
+making use of Miro boards for collecting feedback from the team,
+or looking at the [team github projects board](https://github.com/orgs/ministryofjustice/projects/57/views/6) for planning.
+
+Our main Slack channels are:
+
+#### #data-catalogue
+
+The [#data-catalogue](https://moj.enterprise.slack.com/archives/C04M8224WCV) channel is our team channel,
+where we have discussions about anything ranging from:
+
+- in-depth technical issues
+- questions or discussions around the service
+- to request someone else from the team's help
+- when we're away for lunch or some other appointment
+- general musings
+
+If you can't make standup, post your standup update here.
+
+### GitHub
+
+Access to Data Catalogue's GitHub repositories and projects is managed in Terraform.
```

Comment:
```suggestion
Access to Data Catalogue's GitHub repositories and projects is managed in Terraform.
Here are some repos we use as a team which you should familarise yourself with:
- https://github.com/ministryofjustice/find-moj-data
- https://github.com/ministryofjustice/data-platform

We use MoJ's AWS hosting services for new services if we can, which are run by service teams:
- https://user-guide.cloud-platform.service.justice.gov.uk/
- https://user-guide.modernisation-platform.service.justice.gov.uk/
```

Reply:
We can throw this into the ways of working in a separate PR - [see AP's WoW](https://technical-documentation.data-platform.service.justice.gov.uk/documentation/team/ways-of-working.html)